ports/qemu-arm/Makefile.test

```diff
@@ -19,6 +21,7 @@ $(BUILD)/firmware-test.elf: $(OBJ_COMMON) $(OBJ_TEST)
 	$(Q)$(SIZE) $@
 
 test: $(BUILD)/firmware-test.elf
-	qemu-system-arm -machine $(BOARD) $(QEMU_EXTRA) -nographic -monitor null -semihosting -kernel $< > $(BUILD)/console.out
+    # qemu hangs (e.g. due to alignment errors).
```

Comment:
Please put comment just above the rule, else it'll be printed as part of the execution of the rule (unless that's what's intended...).  Also maybe something like "timeout is to stop qemu if it hangs"